### PR TITLE
LoadTasks task

### PR DESCRIPTION
**Pull request by splatteredbits**

### CODE REVIEW

File: Test/LoadTasks.Tests.ps1 [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+        Invoke-WhiskeyTask -TaskContext $context -Name 'LoadTasks' -Parameter $parameter
+    }
+    catch
+    {
+        Write-Error -ErrorRecord $_
+        $script:failed = $true
+    }
+}
+
+Describe 'LoadTasks' {
```
Comment (jlariono) [inline Test/LoadTasks.Tests.ps1]: What happens when you run the `LoadTasks` task multiple times like when given this whiskey.yml?

```
Build:
- Pipeline:
    Name:
      - Tests
      - Package

Init:
- LoadTasks:
    Path: task.ps1

Tests:
- Pipeline:
    Name: Init
- NUnit2: ...

Package:
- Pipeline:
    Name: Init
- ProGetUniversalPackage: ...
```

File: Test/LoadTasks.Tests.ps1 [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+        $failed | Should -BeTrue
+    }
+}
+
+function ThenFile
+{
+    param(
+        $Name,
+        [Switch]
+        $Exists
```
[the remark this answers is not in the document]
Comment (splatteredbits) [reply Test/LoadTasks.Tests.ps1]: It's there to give the tests a more fluent grammar:

```
    ThenFile 'fubar' -Exists
```

File: Whiskey/en-US/about_Whiskey_LoadTasks_Task.help.txt [hunk 3]
```diff
@@ -0,0 +1,47 @@
+TOPIC
+    about_Whiskey_LoadTasks_Task
```
[the remark this answers is not in the document]
Comment (splatteredbits) [reply Whiskey/en-US/about_Whiskey_LoadTasks_Task.help.txt]: Fixed.

File: Whiskey/Tasks/Import-WhiskeyTask.ps1 [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    foreach( $path in $paths )
+    {
+        $knownTasks = @{}
+        Get-WhiskeyTask | ForEach-Object { $knownTasks[$_.Name] = $_ }
+        # We do this in a background script block to ensure the function is scoped correctly. If it isn't, it 
+        # won't be available outside the script block. If it is, it will be visible after the script block completes.
+        & {
+            . $path
+        }
+        $newTasks = Get-WhiskeyTask | Where-Object { -not $knownTasks.ContainsKey($_.Name) }
```
[the remark this answers is not in the document]
Comment (splatteredbits) [reply Whiskey/Tasks/Import-WhiskeyTask.ps1]: Fixed.

File: Test/LoadTasks.Tests.ps1 (hunk 1 above)
Comment (splatteredbits) [reply Test/LoadTasks.Tests.ps1]: Fixed.

**Comment by splatteredbits** [comment]
@jlariono Ready for re-review.

**Pull request merged**